src/prommis/solvent_extraction/solvent_extraction.py

```diff
@@ -193,6 +193,7 @@ def param_init(b, s, k, l, m):
             self.mscontactor.stream_component_interactions,
             initialize=param_init,
             mutable=True,
+            domain=Any,
```

Comment:
I do not think you should be using the `Any` domain - it is somewhat meaningless and I think it might trigger Pyomo warnings. If you need a domain, use `Reals` or `NonNegativeReals`.

Reply:
If the domain is anything but `Any`, I see this error when trying to run any flowsheet with SX: 
![image](https://github.com/prommis/prommis/assets/96305519/07b89a50-c8e4-499c-9464-c0637309f676)

Is the solution to this error specifying initial default values for each of the partition coefficients?

Reply:
This should have `initialize=1` rather than calling a function that just assigns a value of 1 to any index. This should fix the issue.

@ArkoprabhoDasgupta The `initialize` argument needs to take either a single value, a `dict` with values for important keys, or an iterator - the function you provided does not actually return anything.

Reply:
Thanks, having the following works:
`        self.partition_coefficient = Param(
            self.mscontactor.elements,
            self.mscontactor.stream_component_interactions,
            initialize=1,
            mutable=True,
            doc="The fraction of component that goes from aqueous to organic phase",
        )`